packages/next-server/package.json

```diff
@@ -48,9 +47,11 @@
     "@babel/plugin-proposal-class-properties": "7.1.0",
     "@babel/plugin-proposal-object-rest-spread": "7.0.0",
     "@babel/plugin-syntax-dynamic-import": "7.0.0",
+    "@babel/plugin-transform-classes": "7.1.0",
```

Comment:
What is this needed for?

Reply:
@timneutkens will get a `TypeError: Class constructor Server cannot be invoked without 'new'` error when running tests

Reply:
Sure, I'm checking the tests 👌

Reply:
Since we are not transforming `class Server` to function (in next-server/next-server.js), but we have `class DevServer extends Server` transformed (in next/server/next-dev-server.js`) so it's the cause.

I'm thinking about using babel env `node > 8` for files under next/server, what do you say?

Reply:
Yes you can implement the same change for next/server too 👍

Reply:
Done 👌